src/features/commands/character/character-command.service.ts

```diff
@@ -16,7 +17,8 @@ export class CharacterCommandService {
   public message(message: Message): void {
     CharacterService.getInstance().getEntity(message.author.id)
     .then((characterFound) => {
-      DisplayMessageService.getInstance().displayMessage(message, `Your character's name is: ${characterFound.name}`)
+      const boldCharacterName = MessageFormattingBoldererService.getInstance().makeItBold(characterFound.name)
```

Comment:
just `bold` as a method name is enough. Agree?

Reply:
Yes.